tests/unit/device.ts

```diff
@@ -806,6 +806,20 @@ describe('Device', function() {
           assert.equal(device['_preferredURI'], ['wss://voice-js.dublin.twilio.com/signal']);
         });
 
+        context('when chunderw is set', () => {
+          it('should use chunderw as the preferred uri if it is a string', () => {
+            device['_options'].chunderw = 'foo';
+            pstream.emit('connected', { region: 'EU_IRELAND', edge: Edge.Dublin });
+            assert.equal(device['_preferredURI'], ['wss://foo/signal']);
+          });
+
+          it('should use the first chunderw as the preferred uri if it is an array', () => {
+            device['_options'].chunderw = ['foo', 'bar'];
+            pstream.emit('connected', { region: 'EU_IRELAND', edge: Edge.Dublin });
+            assert.equal(device['_preferredURI'], ['wss://foo/signal']);
+          });
+        });
+
```

Comment:
I see `_getChunderws(): string[] | null` , do you want to add testcase for `chunderw = null` case?

Reply:
`null` is the default value. It's already covered by the two tests above.